server/testmock/database.go

```diff
@@ -67,8 +93,9 @@ func getMainDatabaseConnection(container *gnomock.Container) (*sql.DB, error) {
 }
 
 func createRandomDatabaseForTest(db *sql.DB, baseDatabase string) (*sql.DB, error) {
-	epoch := time.Now().UnixNano()
```

Comment:
this approach for "randomization" of DB names was causing collisions when running multiple operations in parallel, so I changed it to more random approach